README.md

```diff
@@ -0,0 +1,19 @@
+#network_tools
+
+###This repository holds sample code for a number of network tools implemented with python.
+
+
+*Feb 10th, 2015* - Added a echo server and client with test functionality in Python. 
+                   This server will reprint what the client sent and then will give back 
+                   the message to the server telling it what it said.
+
+
+Collaborations:
+Nick Becker
+
+
+Resources:
+<ul>
+<li>Cris Ewing</li>
+<li>Nick Draper</li>
+<li>Ben Friedland</li>
```

Comment:
Can you make a bulleted list work in markdown format?  

* Like
* This 
* One 
* Here?